modules/post-install-extend-existing-to-zone-type.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/aws-compute-edge-zone-tasks.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="post-install-existing-local-zone-subnet_{context}"]
+= Create network requirements in an existing VPC that uses AWS Local Zones or Wavelength Zones
+
+If you want a Machine API to create an Amazon EC2 instance in a remote zone location, you must create a subnet in a {zone-type} location. You can use any provisioning tool, such as Ansible or Terraform, to create subnets in the existing Virtual Private Cloud (VPC).
+
+You can configure the CloudFormation template to meet your requirements. The following subsections include steps that use CloudFormation templates to create the network requirements that extend an existing VPC to use an AWS {zone-type}.
+
+For Local Zones, on completing these steps in the following subsections, you create the following resources:
+
+Extending nodes to Local Zones requires that you create the following resources:
```

Comment:
```suggestion
* Extending nodes to Local Zones requires that you create the following resources:
```
If these are created resources with subsections, they should be bulleted and sub-bulleted.